superset/db_engine_specs/base.py

```diff
@@ -441,17 +441,6 @@ def set_or_update_query_limit(cls, sql: str, limit: int) -> str:
         parsed_query = sql_parse.ParsedQuery(sql)
         return parsed_query.set_or_update_query_limit(limit)
 
-    @staticmethod
-    def excel_to_df(**kwargs: Any) -> pd.DataFrame:
-        """ Read excel into Pandas DataFrame
-           :param kwargs: params to be passed to DataFrame.read_excel
-           :return: Pandas DataFrame containing data from excel
-        """
-        kwargs["encoding"] = "utf-8"
-        kwargs["iterator"] = True
```

Comment:
`encoding` and `iterator` are not supported on `read_excel`: 
https://pandas.pydata.org/docs/reference/api/pandas.read_excel.html#pandas.read_excel